README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/mozilla/api.webmaker.org.svg)](https://travis-ci.org/mozilla/api.webmaker.org)
+
 # api.webmaker.org
```

Comment:
Would be nice to add some docs here for:
- How to install
- How to configure
- How to run
- How to test